posthog/temporal/common/posthog_client.py

```diff
@@ -12,30 +15,56 @@
 
 class _PostHogClientActivityInboundInterceptor(ActivityInboundInterceptor):
     async def execute_activity(self, input: ExecuteActivityInput) -> Any:
-        ph_client = Client(api_key="sTMFPsFhdP1Ssg", enable_exception_autocapture=True)
-
         try:
-            activity_result = await super().execute_activity(input)
-        except:
-            raise
-        finally:
-            ph_client.flush()
+            return await super().execute_activity(input)
+        except Exception as e:
+            activity_info = activity.info()
+            properties = {
+                "temporal.execution_type": "activity",
+                "module": input.fn.__module__ + "." + input.fn.__qualname__,
+                "temporal.activity.attempt": activity_info.attempt,
+                "temporal.activity.id": activity_info.activity_id,
+                "temporal.activity.type": activity_info.activity_type,
+                "temporal.activity.task_queue": activity_info.task_queue,
+                "temporal.workflow.id": activity_info.workflow_id,
+                "temporal.workflow.namespace": activity_info.workflow_namespace,
+                "temporal.workflow.run_id": activity_info.workflow_run_id,
+                "temporal.workflow.type": activity_info.workflow_type,
+            }
+            if len(input.args) == 1 and is_dataclass(input.args[0]):
+                team_id = getattr(input.args[0], "team_id", None)
+                if team_id:
+                    properties["team_id"] = team_id
```

Comment:
Could we actually add properties for all input vars please? Will be super useful for debugging (seeing whether `reset_pipeline=True` or seeing the `source_id`)

Reply:
Some of them may be sensitive though...

Reply:
But if we have a way of excluding/redacting any sensitive values, then this sounds like a good suggestion :+1:

Reply:
hmm yeah, I was thinking of adding those but wasn't sure if they could be sensitive like Tomas said. I can have a think if there's a way to get something working here once this has been merged